package.json

```diff
@@ -606,10 +569,6 @@
                     "command": "makefile.launchRun",
                     "when": "makefile:localRunFeature"
                 },
-                {
-                    "command": "makefile.setBuildConfiguration",
-                    "when": "makefile:fullFeatureSet"
-                },
                 {
```

Comment:
why remove this block?

Reply:
Removing this means that the command pallete "Set Build Configuration" is no longer gated by the "makefile:fullFeatureSet" context variable. Ie, even though the current configuration is impaired in some way (eg, no Makefile in the current makeDirectory), the user can always switch to another configuration.

It makes the "Set Build Configuration" command always available, as long as the extension is activated.

Reply:
Yes, somehow I thought it was removed completely. I forgot to expand and look what is the block that was part of. Thank you. Yes good change.